README.md

```diff
@@ -8,6 +8,7 @@
 
 * [Requirements](#requirements)
 * [Installation](#installation)
+  + [Swift Package Manager](#swift-package-manager)
```

Comment:
Can you put Swift Package Manager section below CocoaPods, since the second one is more popular so far.

Reply:
Done